wondrous-app/components/Pod/members/FilteredRoles.tsx

```diff
@@ -0,0 +1,24 @@
+import RolePill from 'components/Common/RolePill';
+import { Role } from 'types/common';
+import { FilteredRolesContainer } from './styles';
+
+type Props = {
+  roles: Role[];
+  selectedRoleIds: string[];
+};
+
+const FilteredRoles = (props: Props) => {
+  const { roles, selectedRoleIds } = props;
+
+  const selectedRoles = roles?.filter((role) => selectedRoleIds.includes(role.id));
```

Comment:
let's use useMemo here, it's an expensive operation

`const selectedRoles = useMemo(() => roles?.filter((role) => selectedRoleIds.includes(role.id)), [roles, selectedRoleIds])`